src/main/java/bio/terra/workspace/db/WorkspaceDao.java

```diff
@@ -44,4 +45,25 @@ public boolean deleteWorkspace(UUID workspaceId) {
         jdbcTemplate.update("DELETE FROM workspace WHERE workspace_id = :id", paramMap);
     return rowsAffected > 0;
   }
+
+  public WorkspaceDescription getWorkspace(UUID workspaceId) {
+    String sql = "SELECT * FROM workspace where workspace_id = (:id)";
+
+    Map<String, Object> paramMap = new HashMap<>();
+    paramMap.put("id", workspaceId.toString());
+
+    Map<String, Object> queryOutput = jdbcTemplate.queryForMap(sql, paramMap);
```

Comment:
I realize I probably set a bad example on this, but could you create a new ErrorReport-based exception type to throw here, rather than returning JDBC's native EmptyResultDataAccessException?